bots/images/scripts/virt-builder-fedora

```diff
@@ -56,7 +56,7 @@ BASE=$(dirname $0)
 out=$1
 os=$2
 arch=$3
-test -z "$BOOTSTRAP_VOLUME_SIZE" && BOOTSTRAP_VOLUME_SIZE="8G"
+test -z "$BOOTSTRAP_VOLUME_SIZE" && BOOTSTRAP_VOLUME_SIZE="15G"
```

Comment:
I think we only want to change this for openshift. Can we set the env variable only for the the openshift builder.

Reply:
done